apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -77,10 +135,12 @@ export class Utilities {
    * Retries a function until a timeout is reached. The function is expected to throw if it failed and
    *  should be retried.
    */
-  public static retryUntilTimeout<TResult>(fn: () => TResult,
-                                           maxWaitTimeMs: number,
-                                           getTimeoutError: (innerError: Error) => Error,
-                                           fnName: string): TResult {
+  public static retryUntilTimeout<TResult>(
+    fn: () => TResult,
+    maxWaitTimeMs: number,
+    getTimeoutError: (innerError: Error) => Error,
+    fnName: string
```

Comment:
We should use an English word.  `fn` is not a word.

Reply:
`function` is a reserved word. `fn` is a pretty widely understood abbreviation, so I'd rather leave this as-is.

Reply:
I disagree.  People who like to save typing by writing unintelligible code can argue that all sorts of jiberrish is a "widely understood abbreviation".  Whereas our convention is to use English words, and the English dictionary says:  https://www.dictionary.com/browse/fn

Also, why are we reporting function names anyway?  That's not meaningful to an end user.  The parameter should really be something like "operationDescription" and then the logging should be like:

```ts
  // "Stalled for 1.2 seconds while deleting lock file"
  // "Stalled for 1.2 seconds while renaming folder"
  // ...
  console.log(`Stalled for ${totalSeconds} seconds during ${operationDescription}`)
```

Admittedly this is out of scope for the current PR, but you guys went end edited that line, so you invited this pedantry.  😁

Reply:
The logging thing is weird. I'll add a comment that we should revisit that.